src/main/java/com/algorand/algosdk/v2/client/model/Account.java

```diff
@@ -0,0 +1,110 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.algorand.algosdk.v2.client.common.PathResponse;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Account information at a given round. 
+ * Definition: 
+ * data/basics/userBalance.go : AccountData 
+ * 
+ */
+public class Account extends PathResponse {
+
+	/**
+	 * the account public key 
+	 */	@JsonProperty("address")
```

Comment:
Looks like there's still a newline issue here, `@JsonProperty` should be on the next line.

Reply:
I don't want to add another extra line. This is intentional.